src/registrar/assets/js/get-gov.js

```diff
@@ -209,10 +206,31 @@ function handleInputValidation(e) {
 function handleValidationClick(e) {
   const attribute = e.target.getAttribute("validate-for") || "";
   if (!attribute.length) return;
+
   const input = document.getElementById(attribute);
   runValidators(input);
 }
 
+
+function handleFormsetValidationClick(e) {
+  // Check availability for alternative domains
+  
+  const alternativeDomainsAvailability = document.getElementById('check-avail-for-alt-domains');
+  
```

Comment:
(Optional) I would suggest having "alternativeDomainsAvailability" be a variable that you pass in, since it is already defined in the IIFE, something like `function handleFormsetValidationClick(e,  alternativeDomainsAvailability)` or `function handleFormsetValidationClick(e,  availabilityButton)`. This would make it easier in the future to maintain if for example the id of that button needs to change